src/main/java/com/igot/cb/metrics/MetricsComponent.java

```diff
@@ -0,0 +1,44 @@
+package com.igot.cb.metrics;
+
+import com.igot.cb.pores.util.Constants;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Slf4j
+@Service
+public class MetricsComponent {
+
+    private int operationCount = 0;
+    private long totalTimeTaken = 0;
+    private long operationStartTime = 0;
+
+    public void startOperation() {
+        if (operationStartTime != 0) {
+            throw new IllegalStateException("An operation is already in progress.");
+        }
+        operationStartTime = System.currentTimeMillis();
+    }
+
+    public void endOperation(String operationName) {
+        if (operationStartTime == 0) {
+            throw new IllegalStateException("No operation is currently in progress.");
+        }
+        long elapsedTime = System.currentTimeMillis() - operationStartTime;
+        totalTimeTaken += elapsedTime;
+        operationCount++;
+        operationStartTime = 0;
+        log.info("Operation time taken:: {}: {}",operationName,elapsedTime);
+    }
+
+    public Map<String, Object> getOperationStats() {
+        Map<String, Object> operationData = new HashMap<>();
+        operationData.put(Constants.OPERATION_COUNT, operationCount);
+        operationData.put(Constants.TIME_TAKEN, totalTimeTaken);
+        operationCount = 0;
+        totalTimeTaken = 0;
+        return operationData;
+    }
+}
```

Comment:
this is not what I've informed... 

you need to keep this at server level only -- not intended for returning the data on api response... 
I mentioned count and value to be incremented at the component level... 

you provide new rest api to get the metrics which simply returns only the map details in the component...